app/src/main/java/com/android/solvit/seeker/ui/service/Services.kt

```diff
@@ -71,15 +73,27 @@ fun ServicesScreen(
     navigationActions: NavigationActions,
     listProviderViewModel: ListProviderViewModel
 ) {
+
+  val lpvm by listProviderViewModel.providersList.collectAsState()
```

Comment:
Should we use a more understandable variable name such as simply "providers"?

Reply:
It was just for debugging purposes I will in fact clean variable names